.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,14 +8,14 @@ Please fill out the information below to expedite the review and (hopefully)
 merge of your pull request!
 -->
 
-<!-- What changes are being made? (What feature/bug is being fixed here?) -->
+<!-- What changes are being made? (What feature/bug is being addresses here?) -->
 **What**:
 
-<!-- Why are these changes necessary? -->
-**Why**:
+<!-- Please provide a link to your fork's Storybook. See README notes on how to do this. -->
+**Link to Storybook**:
 
-<!-- How were these changes implemented? -->
-**How**:
+<!-- Are there any upstream issues or separate issues you need to reference? -->
+**Additional issues**:
```

Comment:
I'm wondering if it makes sense to add this in addition to the issues referenced from the `npm run commit` process that identifies them. Will this create duplicate referencing? It may be fine, just a thought.

Reply:
@danseethaler good question... My understanding is that adding "Closes # X" within `npm run commit` OR adding "Closes # X" in the PR notes will close the issue. I guess my preference would be to include "Closes" in the PR comments (under the first question) instead of the commit b/c there are certain cases we might not close an issue, but still merge a commit (so you'd have flexibility to change it at any point). I'm flexible on this though (and think either is OK). Thoughts?

I made a note about additional issues at the end for a few things that you might list:

* other upstream issues / PRs affected by this PR
* other relevant issues that are related to this PR (but not closed by this PR)

I'm quite flexible on this and curious what others think...

Reply:
Yeah, it's also my preference to have the issues reference in the PR description rather than the commit message since it's much more visible when glancing at a PR. I'm mostly wondering if we're asking for it in both places if it will show multiple references in the issue we're closing. Not a huge deal either way.

I think what you've got here is good and we see how it goes 👍🏼